GTFO-API/Patches/GearManager_Patches.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using Gear;
+using HarmonyLib;
+using Player;
+using UnhollowerBaseLib.Attributes;
+
+namespace GTFO.API.Patches
+{
+    [HarmonyPatch(typeof(GearManager))]
+    internal class GearManager_Patches
+    {
+        [HideFromIl2Cpp]
+        public static Dictionary<InventorySlot, string[]> BotFavorites { get; set; }
+
+        [HarmonyPatch(nameof(GearManager.Setup))]
+        [HarmonyWrapSafe]
+        [HarmonyPostfix]
+        public static void Setup_Postfix()
+        {
+            BotFavorites = new Dictionary<InventorySlot, string[]>
+            {
+                { InventorySlot.GearMelee, new string[4] },
+                { InventorySlot.GearStandard, new string[4] },
+                { InventorySlot.GearSpecial, new string[4] },
+                { InventorySlot.GearClass, new string[4] },
+                { InventorySlot.HackingTool, new string[4] }
+            };
+
+            GearManager.FavoritesData = new GearFavoritesData();
```

Comment:
Omit type names and initialize the dictionary by key to value
```suggestion
            BotFavorites = new()
            {
                [InventorySlot.GearMelee] = new string[4],
                [InventorySlot.GearStandard] = new string[4],
                [InventorySlot.GearSpecial] = new string[4],
                [InventorySlot.GearClass] = new string[4],
                [InventorySlot.HackingTool] = new string[4]
            };

            GearManager.FavoritesData = new();
```